server/WebAPI/DTO/Trip/TripRequestDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace WebAPI.DTO.Trip
+{
+  public class TripRequestDTO
+  {
+    public int DepartureCountryId { get; set; }
+    public string DepartureCity { get; set; }
+    public DateTime DepartureDate { get; set; }
+
+    public int ArrivalCountryId { get; set; }
+    public DateTime ArrivalDate { get; set; }
+
+    public decimal TotalPrice { get; set; }
+    public string Currency { get; set; }
+
+    public string FlightTime { get; set; }
+    public string TransplantTime { get; set; }
+    public string DifferenceInTime { get; set; }
```

Comment:
it would be good to add attributes https://docs.microsoft.com/ru-ru/dotnet/api/system.componentmodel.dataannotations?view=netframework-4.8

Reply:
You mean validation

Reply:
yes, validation attributes